scripts/ci-e2e.sh

```diff
@@ -199,5 +199,9 @@ main(){
     [ -z "${BOSKOS_HOST:-}" ] || release_account >> "$ARTIFACTS/logs/boskos.log" 2>&1
 }
 
+if [[ $- =~ x ]]; then
```

Comment:
@Prajyot-Parab , has this been added as an additional check even after adding the snippet in boskos.sh?

Reply:
Yes, to ensure we don't end up exposing anything in future even if boskos is removed/disabled. Also this applies to overall ci-e2e scope itself. (the other one was specifically for boskos)